src/transformStateWithClones.js

```diff
@@ -64,8 +64,34 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const arrOfStates = [];
+  let copyOfStateObj = Object.assign({}, state);
```

Comment:
```suggestion
  let copyOfStateObj = { ...state };
```
You could use spread-operator here